yt_dlp/extractor/anvato.py

```diff
@@ -234,8 +232,8 @@ def _get_video_json(self, access_key, video_id, extracted_token):
         server_time = self._server_time(access_key, video_id)
         input_data = f'{server_time}~{md5_text(video_data_url)}~{md5_text(server_time)}'
 
-        auth_secret = intlist_to_bytes(aes_encrypt(
-            bytes_to_intlist(input_data[:64]), bytes_to_intlist(self._AUTH_KEY)))
+        auth_secret = bytes(aes_encrypt(
+            list(input_data[:64]).encode(), list(self._AUTH_KEY)))
```

Comment:
still wrong, need to ecode the data being cast to list, not encode the list
```suggestion
            list(input_data[:64].encode()), list(self._AUTH_KEY)))
```